### DISCUSSION TITLE
Tabs for relation

### DISCUSSION THREAD

**Discussion by SerhatReis** [twentyhq/twenty#7123] [Ideas]
It would be great if there was an option to display the relationship of an object in a separate tab, along with the associated data. For example, 'Order' with a tab for 'Order Lines.'

![image](https://github.com/user-attachments/assets/ad64ce46-61c6-4854-a37a-749121bbed55)

**Comment by pgarratt** [comment]
I second that, an ability to add custom pages as sub list from a main entity would be huge. could be used for all sorts of things. 
Order - Order Lines, user - audit trail, product - product groups, company - related assets, etc..
Also helpful would be a way to open a specific sub item to view / edit the data.

**Comment by Bonapara** [comment]
We will do that when we'll introduce custom layouts! (around Q4 25 - Q1 26)

**Reply by BOHEUS** [reply]
@Bonapara is it included in ongoing implementation of custom layouts of record details?

**Comment by Bonapara** [comment]
Yes! You will have the ability to pin relations anywhere
